extension/src/time_weighted_average.rs

```diff
@@ -422,6 +488,29 @@ mod tests {
             let stmt = "SELECT average(time_weight('LOCF', ts, val)) FROM test";
             assert!((select_one!(client, stmt, f64) - 10.0).abs() < f64::EPSILON);
 
+            let stmt = "SELECT first_val(time_weight('LOCF', ts, val)) FROM test";
+            assert!((select_one!(client, stmt, f64) - 10.0).abs() < f64::EPSILON);
+            let stmt = "SELECT last_val(time_weight('LOCF', ts, val)) FROM test";
+            assert!((select_one!(client, stmt, f64) - 20.0).abs() < f64::EPSILON);
+
+	    // arrow syntax should be the same
```

Comment:
misaligned comments; maybe you've inserted TAB characters?

Reply:
Thanks for catching... adding a `(setq indent-tabs-mode nil)` rust-mode-hook as we speak